homeassistant/components/bluesound/media_player.py

```diff
@@ -401,34 +386,56 @@ async def async_update_status(self) -> None:
             )
             raise
 
+    async def update_sync_status(self) -> None:
+        """Update the internal status."""
+        etag = None
+        if self._sync_status:
+            etag = self._sync_status.etag
+        sync_status = await self._player.sync_status(
+            etag=etag, poll_timeout=POLL_TIMEOUT, timeout=POLL_TIMEOUT + 5
+        )
+
+        self._sync_status = sync_status
+
+        if sync_status.master is not None:
+            self._is_master = False
+            master_id = f"{sync_status.master.ip}:{sync_status.master.port}"
+            master_device = [
```

Comment:
Can we migrate this naming to `main` and `secondary`?

Reply:
That's on my TODO list :+1: